website/src/pages/roadmap.md

```diff
@@ -4,69 +4,55 @@ last_modified_at: 2019-12-30T15:59:57-04:00
+| Efficient bootstrap and migration of existing non-Hudi dataset | 0.13 | [HUDI-1265](https://issues.apache.org/jira/browse/HUDI-1265) |
+| Eliminating physical partitioning with efficient logical partitioning and file pruning | 0.13 | [HUDI-512](https://issues.apache.org/jira/browse/HUDI-512) |
+| Lock-Free message queue to improve writing efficiency | 0.13 | [RFC-53](https://github.com/apache/hudi/blob/master/rfc/rfc-53/rfc-53.md), [HUDI-3963](https://issues.apache.org/jira/browse/HUDI-3963) |
+
+## Programming APIs
+
+|Feature|Target Release|Tracking|
+|------------|--------|-----------|
+| Redesign and optimization of record payload abstraction | 0.13 | [RFC-46](https://github.com/apache/hudi/blob/master/rfc/rfc-46/rfc-46.md), [HUDI-3217](https://issues.apache.org/jira/browse/HUDI-3217) |
+| Optimized storage layout for cloud object stores | 0.13 | [RFC-60](https://github.com/apache/hudi/pull/5113), [HUDI-3625](https://issues.apache.org/jira/browse/HUDI-3625) |
+| Support of verification with multiple event_time fields | 0.13 | [RFC-59](https://github.com/apache/hudi/pull/6382), [HUDI-4569](https://issues.apache.org/jira/browse/HUDI-4569) |
 
 ## Execution Engine Integration
 
 |Feature|Target Release|Tracking|
 |------------|--------|-----------|
-|Spark SQL DML fixes & enhancements |0.11|[HUDI-1658](https://issues.apache.org/jira/browse/HUDI-1658)|
-|Data-skipping for Hive and Spark based on col_stats from metadata table  |0.11|[HUDI-1296](https://issues.apache.org/jira/browse/HUDI-1296), [RFC-27](https://github.com/apache/hudi/pull/4280)|
-|Non-keyed tables with updates and deletes |0.11|[HUDI-2968](https://issues.apache.org/jira/browse/HUDI-2968)|
-|Trino Connector for Hudi, with read/query support  |0.12|[HUDI-2687](https://issues.apache.org/jira/browse/HUDI-2687), [RFC-38](https://github.com/apache/hudi/pull/3964)|
-|Spark Datasource V2|0.12|[HUDI-1297](https://issues.apache.org/jira/browse/HUDI-1297) ,[HUDI-2531](https://issues.apache.org/jira/browse/HUDI-2531)|
-|Complete ORC Support across query engines |0.12|[HUDI-57](https://issues.apache.org/jira/browse/HUDI-57)|
-|Presto Connector for Hudi, with read/query support |0.12|[PRESTO-17006](https://github.com/prestodb/presto/issues/17006)|
-|Multi-Modal indexing full integration across Presto/Trino/Spark queries |0.12 -> 1.0.0|[HUDI-1822](https://issues.apache.org/jira/browse/HUDI-1822)|
-|Materialized Views with incremental updates using Flink |1.0.0||
-|SQL DML support for Presto/Trino connectors (could be accelerated based on community feedback) |1.0.0||
-|Explore other execution engines/runtimes (Ray, native Rust, Python) |1.0.0||
+| Spark datasource V2 read | 0.13 | [HUDI-4449](https://issues.apache.org/jira/browse/HUDI-4449) |
+| Upgrade to Spark 3 as the default profile | 0.13 | [HUDI-3431](https://issues.apache.org/jira/browse/HUDI-3431) |
```

Comment:
minor: do you want to put the full version tag like 0.13.0 ?

Reply:
Makes sense.  Fixed.